share/instruments/generate_tsv.py

```diff
@@ -51,8 +50,8 @@ def find_text(element, tagName, default='[null]'):
     tag = element.find(tagName)
     return default if tag is None else tag.text
 
-os.chdir(os.path.dirname(os.path.realpath(__file__))) # make all paths relative to this script's directory
-os.makedirs('tsv', exist_ok=True)
+script_dir = Path(__file__).parent
+Path.mkdir(script_dir / 'tsv', exist_ok=True)
```

Comment:
Shouldn't all relative paths used throughout the script be updated like this?

Reply:
updated.

Reply:
But what about line 57 (`root = ET.parse('instruments.xml').getroot()`) and 60/61 (`load_translations('../locale/instruments_it.ts')`)?